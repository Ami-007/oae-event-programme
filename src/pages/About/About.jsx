import React from "react";
import Navbar from "../../components/Navbar/Navbar";
import Photo from "../../components/Photo/Photo";

const About = () => {
  return (
    <div>
      <h1>About page</h1>
      <Photo
        imageSrc="https://place-puppy.com/300x300" alt="puppy"
        description="A picture of a man's best friend"/>
      <Navbar/>
    </div>
  );
};

export default About;
