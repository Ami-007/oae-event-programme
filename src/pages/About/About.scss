@use "../../assets/sass/variables" as *;

.about {
  &__content {
    padding: 0 2.5rem;
  }
  &__callout {
    color: var(--theme-color-text);
    font-family: $modern-header;
    display: flex;
    justify-content: flex-end;
    padding: 0 2.5rem 0 2.5rem;
  }
}
