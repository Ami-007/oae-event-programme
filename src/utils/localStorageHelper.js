export const getDarkTheme = () => {
  //   if (darkTheme == true) {
  //     localStorage.setItem("--background-color", darkTheme);
  //   } else {
  //     localStorage.setItem("theme", "light");
  //   }
  console.log(localStorage.getItem("darkMode"));

  // convert isDarkMode to a Boolean
  // return that boolean

  return true;
};

export const setDarkTheme = (isDarkMode) => {
  localStorage.setItem("darkMode", isDarkMode.toString());
};
