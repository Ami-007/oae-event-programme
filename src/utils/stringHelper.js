export const getFontType = (fontType) => {
  if (typeof fontType === "string") {
    fontType = parseInt(fontType);
  }
  if (fontType === 1) {
    return "modern";
  } else if (fontType === 2) {
    return "classic";
  } else {
    return "modern";
  }
};

export default getFontType;
