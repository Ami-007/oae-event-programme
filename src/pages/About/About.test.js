import { customRender } from "../../utils/testUtils";
import About from "./About";

test("Renders photo component in the About page", () => {
  const { container } = customRender(<About />);
  expect(container).toMatchSnapshot();
});
