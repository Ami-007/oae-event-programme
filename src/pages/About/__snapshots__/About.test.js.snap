// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders photo component in the About page 1`] = `
<div>
<<<<<<< HEAD
  <div>
    <div
      class="page-header"
    >
      <div
        class="header-home"
      >
        <h1
          class="header-home__title"
        >
          our story
        </h1>
        <span
          class="header-home__subtitle"
        >
          30 Years of Breaking The Rules
        </span>
      </div>
      <div
        class="event-header-details"
      >
        <p
          class="event-header-details__time-date"
        />
        <p
          class="event-header-details__venue-location"
        >
          <span
            class="event-header-details__venue"
          />
           
        </p>
      </div>
      <div
        class="header-image-video-container"
      >
        <iframe
          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
          allowfullscreen=""
          class="youtube-embed"
          frameborder="0"
          src="https://www.youtube.com/embed/aString"
          title="Embedded youtube"
        />
      </div>
    </div>
    <div
      class="photo"
    >
      <img
        alt=""
        class="photo__image"
        src="image.png"
      />
      <span
        class="photo__description"
      >
        -
      </span>
=======
  <div
    class="layout modern"
  >
    <div>
      <h1>
        About page
      </h1>
      <div
        class="photo"
      >
        <img
          alt="puppy"
          class="photo__image"
          src="https://place-puppy.com/300x300"
        />
        <span
          class="photo__description"
        >
          A picture of a man's best friend
        </span>
      </div>
      <div
        class="orchestra-quote"
      >
        <q
          class="orchestra-quote__content"
        >
          COMPLIANCE IS NOT HEALTHY. WHAT I WANT IS A DIALOGUE.
        </q>
        <p
          class="orchestra-quote__author"
        >
          -
          Vladmir Jurowski
        </p>
      </div>
>>>>>>> 121cae44a7869ba7d1d28da4535824af62951851
    </div>
    <nav
      class="navbar"
    >
      <div
        class="navbar__content"
      >
        <a
          class="navbar__schedule-link"
          href="/6bb8ffe8-6e5e-4abc-be7f-b05b749849cf/event-schedule"
        >
          <img
            alt="schedule icon"
            class="navbar__icon"
            src="schedule.svg"
          />
        </a>
        <span
          class="navbar__title"
        >
          SCHEDULE
        </span>
      </div>
      <div
        class="navbar__content"
      >
        <a
          class="navbar__home-link"
          href="/6bb8ffe8-6e5e-4abc-be7f-b05b749849cf"
        >
          <img
            alt="home icon"
            class="navbar__icon"
            src="home.svg"
          />
        </a>
        <span
          class="navbar__title"
        >
          HOME
        </span>
      </div>
      <div
        class="navbar__content"
      >
        <a
          class="navbar__about-link"
          href="/6bb8ffe8-6e5e-4abc-be7f-b05b749849cf/our-story"
        >
          <img
            alt="about icon"
            class="navbar__icon"
            src="info.svg"
          />
        </a>
        <span
          class="navbar__title"
        >
          ABOUT
        </span>
      </div>
    </nav>
  </div>
</div>
`;
